src/main/java/io/aiven/kafka/connect/s3/config/S3SinkConfig.java

```diff
@@ -98,6 +99,9 @@ public class S3SinkConfig extends AivenCommonConfig {
 
     public static final String AWS_ACCESS_KEY_ID_CONFIG = "aws.access.key.id";
     public static final String AWS_SECRET_ACCESS_KEY_CONFIG = "aws.secret.access.key";
+    public static final String AWS_CREDENTIAL_PROVIDER_CONFIG = "aws.credential.provider";
```

Comment:
```suggestion
    public static final String AWS_CREDENTIALS_PROVIDER_CONFIG = "aws.credentials.provider";
```
to be aligned with AWS API